tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
+  sourceMaps: "inline",
+  compact: false,
+  whitelist: [
```

Comment:
It sounds like it may be easiest to have this live in the tool, and somehow access the tool version from https://github.com/meteor/react-packages/tree/master/babel-compiler. What do you think?